client/my-sites/people/people-list-item/style.scss

```diff
@@ -3,35 +3,24 @@
 	box-shadow: none;
 	display: flex;
 	overflow: hidden;
-	padding-left: 0;
-	padding-right: 0;
 	position: relative;
 
 	@include breakpoint( '>480px' ) {
-		padding-top: 24px;
-		padding-bottom: 24px;
-	}
-
-	&:nth-of-type( 1 ) {
-		margin-top: -16px;
-
-		@include breakpoint( '>480px' ) {
-			margin-top: -24px;
-		}
+		padding: 24px;
 	}
 
 	&:last-of-type {
 		border-bottom: none;
-		margin-bottom: -16px;
-
-		@include breakpoint( '>480px' ) {
-			margin-bottom: -24px;
-		}
 	}
 
 	.bulk-editing & {
 		padding-left: 40px;
 	}
+
+	&:focus {
+		outline: 2px solid var( --color-primary-light );
+		z-index: 1;
```

Comment:
Do we need to use the `z-index()` Sass function to add a z-index? I'd always assumed that was a requirement based on /devdocs.

Reply:
yup, we should

Reply:
TIL

Reply:
I added the `z-index` function call. It would be good to get another pair of eyes as I've not done this before :)